daemon/config/config.go

```diff
@@ -160,7 +160,9 @@ type CommonConfig struct {
 	ExecRoot              string                    `json:"exec-root,omitempty"`
 	SocketGroup           string                    `json:"group,omitempty"`
 	CorsHeaders           string                    `json:"api-cors-header,omitempty"`
-	ProxyConfig
+
+	// ProxyConfig holds the proxy-configuration for the daemon.
+	ProxyConfig `json:"proxy-config"`
```

Comment:
How about just `proxies`?

I know you reverted some go changes (I assume it was giving this an explicit field name).
I think it's good to give it a proper field name.

Reply:
Proxies works for me (I was wondering if there would be other proxy-related options that could become relevant); can you think of any?

Reply:
Nope.